rbac: simplify and make authorization HTTP/TCP tasks consistent (#4320)
istio/istio.io: content/docs/tasks/security/authz-http/index.md

* Read the [authorization concept](/docs/concepts/security/#authorization).

> * Follow the [Kubernetes quick start](/docs/setup/kubernetes/install/kubernetes/) to install Istio **with the strict mutual TLS profile**.

Reviewer: ```suggestion
* Follow the [Kubernetes quick start](/docs/setup/kubernetes/install/kubernetes/) to install Istio using the **strict mutual TLS profile**.
```

> * Deploy the [Bookinfo](/docs/examples/bookinfo#deploying-the-application) sample application.

Reviewer: ```suggestion
* Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
```

Author: Thanks, updated.

After deploying the Bookinfo application, go to the Bookinfo product page at `http://$GATEWAY_URL/productpage`. On

> the product page, you can see:

Reviewer: ```suggestion
the product page, you can see the following sections:
```

> * The **Book Details** section on the lower left of the page includes book type, number of

Reviewer: ```suggestion
* **Book Details** on the lower left side, which includes: book type, number of
```

  pages, publisher, etc.

> * The **Book Reviews** section on the lower right of the page.

Reviewer: ```suggestion
* **Book Reviews** on the lower right of the page.
```
